launch_pipelines.py

```diff
@@ -0,0 +1,369 @@
+import argparse
+import datetime
+import json
+import logging
+import pydantic
+import uuid
+import yaml
+
+from pathlib import Path
+from seqerakit import seqeraplatform
+from seqerakit.helper import parse_launch_block
+import yaml
+
+## Globals
+# Global UUID for the launch name
+workflow_uuid = str(uuid.uuid4()).replace("-", "")[:15]
+# Global date for the launch name
+date = datetime.datetime.now().strftime("%Y%m%d")
+
+
+class SeqeraKitError(Exception):
```

Comment:
I'll extend the base class in `seqerakit` to include custom exceptions for failure to launch, etc.